xdem/coreg/biascorr.py

```diff
@@ -460,7 +460,7 @@ def __init__(
         | Literal["norder_polynomial"]
         | Literal["nfreq_sumsin"] = "norder_polynomial",
         fit_optimizer: Callable[..., tuple[NDArrayf, Any]] = scipy.optimize.curve_fit,
-        bin_sizes: int | dict[str, int | Iterable[float]] = 100,
+        bin_sizes: int | dict[str, int | Iterable[float]] = 1000,
```

Comment:
Does this link to an issue? I'm sure it's a good update but it's a bit off topic!

Reply:
Those default parameters were a bit arbitrary (one of my PRs 2 years ago) and didn't render well, noticed that in the doc so adjusted it!